src/Crypto/pbc-cffi.lisp

```diff
@@ -121,17 +121,37 @@ THE SOFTWARE.
 
 ;; -----------------------------------------------------------------------
 
-(cffi:define-foreign-library libpbc
- (:darwin #.(concatenate 'string 
-		       (namestring (asdf:system-relative-pathname 'emotiq "../var/local/lib"))
-		       "/libLispPBCIntf.dylib"))
- (:linux #.(concatenate 'string 
-		     (namestring (asdf:system-relative-pathname 'emotiq "../var/local/lib"))
-		     "/libLispPBCIntf.so"))
- (t (:default "libLispPBCIntf"))
- )
-
-(cffi:use-foreign-library libpbc)
+;; The Lisp runtime load might help us do this differently, but explicit
+;; initialization is much easier to understand
+
+(defun load-dev-dlls ()
+  "loads the DLLs (.so and .dylib) at runtime, from pre-specified directories"
+  (cffi:define-foreign-library
+   libpbc
+   (:darwin #.(concatenate 
+	       'string 
+	       (namestring (asdf:system-relative-pathname 'emotiq "../var/local/lib"))
+	       "/libLispPBCIntf.dylib"))
+   (:linux #.(concatenate 
+	      'string 
+	      (namestring (asdf:system-relative-pathname 'emotiq "../var/local/lib"))
+	      "/libLispPBCIntf.so"))
+   (t (:default "libLispPBCIntf"))))
+
+(defun load-production-dlls ()
+  "loads the DLLs (.so and .dylib) at runtime, from the current directory"
+  (cffi:define-foreign-library
+   libpbc
+   (:darwin "libLispPBCIntf.dylib")
+   (:linux  "./libLispPBCIntf.so")
+   (t (:default "libLispPBCIntf"))))
+
+(defun load-dlls()
+  "load the dev or production dlls at runtime"
+  (if (emotiq:production-p)
```

Comment:
This is a problem: we ought to be able to build "cosi-bls" system standalone.  That system does not currently have a dependency on emotiq, and no emotiq package gets created, so if you simply start up fresh and do (asdf:make "cosi-bls"), you get a package error for emotiq package unknown.  The CI is not helping us in that it masks defects in our system definitions: in this case it quickloads emotiq first, so the package happens to be sitting around. For now, rejecting this PR due to this.

Reply:
I addressed this problem with 

https://github.com/emotiq/emotiq/commit/9177c8cd352c634884500238798f0d4cbcc4ac87

@guitarvydas This is the commit I referenced in the Twist.